sdb/commands/linux/slabs.py

```diff
@@ -197,3 +202,44 @@ def pretty_print(self, objs: Iterable[drgn.Object]) -> None:
             table.add_row(row_dict[sort_field], row_dict)
         table.print_(print_headers=self.args.H,
                      reverse_sort=(sort_field not in ["name", "address"]))
+
+
+class SlubCacheWalker(sdb.Walker):
+    """
+    Walk though all allocated entries in a slub cache.
+
+    EXAMPLES
+        Walk through all the objects in the TCP cache:
+
+            sdb> slabs | filter obj.name == "TCP" | slub_cache
```

Comment:
I think that since this is a Walker, one of the examples could use ` | walk`

Reply:
Good idea! Added a test-case too